10/umbraco-cms/extending/key-vault.md

```diff
@@ -33,6 +33,10 @@ dotnet add package Azure.Identity
 
 ### Configuration
 
+{% hint style="info" %}
+You can find the database connection string under the `ConnectionStrings` section in the `appsettings.json` file. For more information, see the [Connection strings settings](../reference/configuration/connectionstringssettings.md) article.
```

Comment:
I believe we in most places point to specific config sections using the entire "path" (see my suggestion).
Not sure if we want to use a colon or just the dots. It's a little different how it's used throughout the docs 😅 

```suggestion
You can find the database connection string under Umbraco:CMS:ConnectionStrings section in the `appsettings.json` file. For more information, see the [Connection strings settings](../reference/configuration/connectionstringssettings.md) article.
```

Reply:
It makes sense to list the entire path 🙌 I believe it's a common practice to use colons when referring to sections within a JSON configuration file.

I've applied the suggestion to all the files.